### DISCUSSION TITLE
Do you still need title keys?

### DISCUSSION THREAD

**Discussion by Alexsilvachief** [skyline-emu/skyline#1347] [Q&A] [answered]
Hello I just got skyline, I heard that you does not need title keys, it automatically gets them, so is it the same for skyline or do you still need to manually retrieve them?

**Comment by hope3x** [comment, marked as answer]
You'll only need prod.keys to play games.

**Reply by Chibuzo2003** [reply]
Pls where can I download Nintendo switch games pls drop the link kk

**Reply by Crytonics** [reply]
No piracy, dump them from your switch.
